package.yaml

```diff
@@ -58,37 +54,55 @@ default-extensions:
   - DeriveLift
   - DeriveTraversable
   - DerivingStrategies
+  - DerivingVia
   - FlexibleContexts
   - FlexibleInstances
   - GADTs
   - GeneralizedNewtypeDeriving
+  - ImportQualifiedPost
```

Comment:
If you follow [our style guide](https://github.com/freckle/guides/blob/88c8fb381b92de464d734bfc281eb4b4d444a568/haskell-style.md#declaring-extensions) and set the language to `GHC2021`, you don't need most of these extensions explicitly enabled.